### PR TITLE
added FSN UBW, Fate Zero and Slime Diaries

### PR DESCRIPTION
**Pull request by Raventhicc**
Mapped FSN UBW to: 
https://anilist.co/anime/19603/Fatestay-night-Unlimited-Blade-Works/ + https://anilist.co/anime/20792/Fatestay-night-Unlimited-Blade-Works-2nd-Season/
Mapped Fate Zero to:
https://anilist.co/anime/10087/FateZero/ +  https://anilist.co/anime/11741/FateZero-Season-2/
Mapped Slime Diaries to:
https://anilist.co/anime/116741/The-Slime-Diaries/

### CODE REVIEW

File: series-tvdb.en.yaml [hunk 1]
```diff
@@ -121,6 +121,21 @@ entries:
         anilist-id: 20845
       - season: 4
         anilist-id: 21379
+  - title: "Fate/Stay night: Unlimited Blade Works"
+    seasons:
+      - season: 1
+        anilist-id: 19603
+        start: -1 # Anilist has 26 Episodes (incldues Special EP) while TVDB has 25 Episodes (excludes Special EP)
```
Comment (Soitora) [inline series-tvdb.en.yaml:128]:
```suggestion
        start: -1 # Anilist has 26 Episodes (includes Special EP) while TVDB has 25 Episodes (excludes Special EP)
```

File: series-tvdb.en.yaml [hunk 2]
```diff
@@ -121,6 +121,21 @@ entries:
...
+      - season: 1
+        anilist-id: 20792
+        start: 13
+  - title: "Fate/Zero"
+    seasons:
+      - season: 1
+        anilist-id: 10087
+      - season: 1
+        anilist-id: 11741
+        start: 13
```
Comment (Soitora) [inline series-tvdb.en.yaml:138]: Does it start at 13? On **AniList**, Season 1 has 13 episodes, which means Season 2 would start on 14?

File: series-tvdb.en.yaml [hunk 3]
```diff
@@ -121,6 +121,21 @@ entries:
...
       - season: 4
         anilist-id: 21379
+  - title: "Fate/Stay night: Unlimited Blade Works"
+    seasons:
+      - season: 1
+        anilist-id: 19603
+        start: -1 # Anilist has 26 Episodes (includes Special EP) while TVDB has 25 Episodes (excludes Special EP)
+      - season: 1
+        anilist-id: 20792
+        start: 13
```
Comment (Soitora) [inline series-tvdb.en.yaml:131]: Does it start at 13? On **AniList**, Season 1 has 13 episodes, which means Season 2 would start on 14?

File: series-tvdb.en.yaml [hunk 4]
```diff
@@ -435,6 +450,10 @@ entries:
         anilist-id: 111321
       - season: 3
         anilist-id: 111322
+  - title: "The Slime Diaries"
```
Comment (Soitora) [inline series-tvdb.en.yaml:453]: On **TheTVDB**, this is "**The Slime Diaries: That Time I Got Reincarnated as a Slime**", only for German and France is it just "**The Slime Diaries**"

File: series-tvdb.en.yaml (hunk 3 above)
Comment (Raventhicc) [reply series-tvdb.en.yaml:131]: S1 starts at episode 0 instead of episode 1 on Anilist.

File: series-tvdb.en.yaml (hunk 2 above)
Comment (Raventhicc) [reply series-tvdb.en.yaml:138]: yea this is my bad

File: series-tvdb.en.yaml (hunk 4 above)
Comment (Raventhicc) [reply series-tvdb.en.yaml:453]: yep again my bad

File: series-tvdb.en.yaml (hunk 3 above)
Comment (Soitora) [reply series-tvdb.en.yaml:131]: Alright that kinda makes sense, sometimes it's confusing how the starting points work

Comment (Raventhicc) [reply series-tvdb.en.yaml:131]: Episode 0 to 12 (total 13 episodes) are in S1. Episode 0 being **Prologue** and Episode 12 being **The Final Decision** where cour 1 ended. Cour 2 starts at Episode 13 **Time of Departure**.

**Pull request merged**